helusers/tests/test_oidc_api_token_authentication.py

```diff
@@ -4,28 +4,46 @@
 
 from helusers.oidc import ApiTokenAuthentication
 
-from .conftest import encoded_jwt_factory, ISSUER1
+from .conftest import ISSUER1, encoded_jwt_factory
 
 
 @pytest.fixture(autouse=True)
 def auto_auth_server(auth_server):
     return auth_server
 
 
-@pytest.mark.django_db
-def test_valid_jwt_is_accepted(rf, unix_timestamp_now):
-    sut = ApiTokenAuthentication()
+@pytest.fixture
+def user_uuid():
+    return uuid.UUID("b7a35517-eb1f-46c9-88bf-3206fb659c3c")
 
-    user_uuid = uuid.UUID("b7a35517-eb1f-46c9-88bf-3206fb659c3c")
 
-    encoded_jwt = encoded_jwt_factory(
+@pytest.fixture
+def jwt_data(user_uuid, unix_timestamp_now):
+    return dict(
         iss=ISSUER1,
         aud="test_audience",
         iat=unix_timestamp_now - 10,
         exp=unix_timestamp_now + 1000,
         sub=str(user_uuid),
     )
 
+
+@pytest.mark.parametrize(
+    "jwt_data_extra",
+    [
+        dict(),
+        dict(amr="something"),
+        dict(amr=["something"]),
+    ],
+)
+@pytest.mark.django_db
+def test_valid_jwt_is_accepted(rf, jwt_data, user_uuid, jwt_data_extra):
+    sut = ApiTokenAuthentication()
+
+    user_uuid = uuid.UUID("b7a35517-eb1f-46c9-88bf-3206fb659c3c")
+
+    encoded_jwt = encoded_jwt_factory(**{**jwt_data, **jwt_data_extra})
```

Comment:
Can have less asterisks

```suggestion
    encoded_jwt = encoded_jwt_factory(**jwt_data, **jwt_data_extra)
```

Reply:
The dict merge is there if you need to replace entries in `jwt_data`. If you unpack without merging and have the same key in both `jwt_data` and `jwt_data_extra`, this will cause an error.

Reply:
Ah, true. I'm concerned the many asterisks makes this a bit hard to read and understand, so would like to reduce the amount. How about 
```suggestion
    jwt_data.update(jwt_data_extra)
    encoded_jwt = encoded_jwt_factory(**jwt_data)
```